CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 ## Next release
 
+## 0.41.0 / 2020-07-29
+
+* [CHANGE] Configmap-reload: Update to v0.4.0 (#3334)
+* [CHANGE] Update prometheus compatibility matrix to v2.19.2 (#3316)
+* [FEATURE] Add Synthetic Probes support (#2832)
+* [FEATURE] Support Prometheus vertical compaction (#3281)
+* [ENHANCEMENT] Propagate Prometheus image version to statefulset (#3358)
+* [ENHANCEMENT] pkg: Instrument resources being tracked by the operator (#3360)
+* [ENHANCEMENT] Add SecretListWatchSelector to reduce memory and CPU footprint (#3355)
+* [ENHANCEMENT] Added support for configuring CA, cert, and key via secret or configmap. (#3249)
+* [ENHANCEMENT] Consolidate image url logic, deprecating `baseImage`, `sha`, and `tag` in favor of `image` field in CRDs. (#3103)
+* [ENHANCEMENT] Support job name in probe (#3318)
+* [ENHANCEMENT] Decouple alertmanager pod labels from selector labels (#3317)
+* [ENHANCEMENT] Fix probe ingress configs (#3312)
```

Comment:
Same as with `[ENHANCEMENT] Support job name in probe`. We are adding support for probes in the same release, so it isn't really an enhancement when looking from user perspective.

Reply:
unified with probes support :+1: